livekit-plugins/livekit-plugins-silero/setup.py

```diff
@@ -49,12 +49,11 @@
     python_requires=">=3.9.0",
     install_requires=[
         "livekit-agents~=0.7",
-        "torch >= 2, < 3",
-        "torchaudio >= 2",
-        "numpy >= 1, < 2",
-        "onnxruntime~=1.17.0",
+        "onnxruntime~=1.18",
+        "numpy~=1.26",
```

Comment:
This prevents installation of numpy>=2, effectively reverting https://github.com/livekit/agents/pull/398/files. Was this pin intentional?